### PR TITLE
Temporarily redirect monthly statistics

### PR DESCRIPTION
**Pull request by elliotcm**
I've left a bunch of now-unused and inaccessible code here because it's being reverted soon.

## Context

The underlying stats code apparently needs changing.

### Before

Screenshot 2022-10-11 at 15 39 11

### After

Screenshot 2022-10-11 at 15 26 00

### CODE REVIEW

**Comment by elliotcm** [comment]
[Create a redirect from the Monthly statistics page to a new page with copy (temporarily)](https://trello.com/c/1kLa1W9h/740-bug-fix-monthly-stats-page-to-say-the-next-cycles-data-will-be-there-next-month)

**Review by carlosmartinez — APPROVED** [review]
great, thanks

File: app/views/publications/monthly_statistics/temporarily_unavailable.html.erb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<%= content_for :title, 'Monthly Statistics - ITT2023' %>
+
+<p class="govuk-body">
```
Comment (frankieroberto) [inline app/views/publications/monthly_statistics/temporarily_unavailable.html.erb:3]: Can we wrap the whole page in these HTML tags to limit the line width please (makes it easier to read):

```html
<div class="govuk-grid-row">
  <div class="govuk-grid-column-two-thirds">

  [...]  

  </div>
</div>
```

File: app/views/publications/monthly_statistics/temporarily_unavailable.html.erb [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  The ITT2022 recruitment cycle has now finished. The new ITT recruitment cycle started on Tuesday 11 October when Apply for teacher training opened for applications.
```
Comment (frankieroberto) [inline app/views/publications/monthly_statistics/temporarily_unavailable.html.erb:4]:
```suggestion
  The 2021 to 2022 teacher training recruitment cycle has now finished. The 2022 to 2023 recruitment cycle started on Tuesday 11 October when Apply for teacher training opened for applications.
```

File: app/views/publications/monthly_statistics/temporarily_unavailable.html.erb [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+<p class="govuk-body">
+  The first publication of ITT statistics for the new cycle will be on Monday 28 November 2022.
+</p>
+
+<p class="govuk-body">
+  You can access the monthly data extracts from the last cycle (from October 2021 until September 2022) here:
+  <%= govuk_link_to 'https://www.gov.uk/government/publications/monthly-statistics-on-initial-teacher-training-itt-recruitment',
+                    'https://www.gov.uk/government/publications/monthly-statistics-on-initial-teacher-training-itt-recruitment' %>
+</p>
```
Comment (frankieroberto) [inline app/views/publications/monthly_statistics/temporarily_unavailable.html.erb:15]:
```suggestion
  You can view  <%= govuk_link_to 'monthly statistics on initial teacher training', 'https://www.gov.uk/government/publications/monthly-statistics-on-initial-teacher-training-itt-recruitment' %> from the last cycle (from October 2021 until September 2022).</p>
```

**Review by frankieroberto — CHANGES_REQUESTED** [review]

File: app/views/publications/monthly_statistics/temporarily_unavailable.html.erb (hunk 1 above)
Comment (elliotcm) [reply app/views/publications/monthly_statistics/temporarily_unavailable.html.erb:3]: Yep

**Comment by elliotcm** [comment]
@frankieroberto The copy was straight from the google doc in the trello ticket, could you confirm with the relevant stakeholders that these changes are OK?  If so yep no worries.

**Comment by elliotcm** [comment]
Copy changes approved (sight unseen) by @johndoates

**Review by frankieroberto — APPROVED** [review]

File: app/views/publications/monthly_statistics/temporarily_unavailable.html.erb [hunk 4]
```diff
@@ -0,0 +1,22 @@
+<%= content_for :title, 'Monthly Statistics - ITT2023' %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
```
Comment (frankieroberto) [inline app/views/publications/monthly_statistics/temporarily_unavailable.html.erb:4]: One last thing, sorry - can we add an h1 please?
```suggestion
  <div class="govuk-grid-column-two-thirds">
  
  <h1 class="govuk-heading-l">Monthly statistics</h1>
```

**Review by frankieroberto — CHANGES_REQUESTED** [review]

**Review by frankieroberto — APPROVED** [review]

**Pull request merged**